prody/proteins/interactions.py

```diff
@@ -1774,8 +1775,8 @@ def calcLigandInteractions(atoms, **kwargs):
 
         return Ligands, analyzedLigand
 
-    except:
-        LOGGER.info("Install Openbabel and PLIP.")
+    except ImportError:
```

Comment:
again there's quite a bit that could happen besides the import error. If it's the import error that you want to catch then you should handle it right after trying the import

Reply:
Done.